web/Makefile

```diff
@@ -14,33 +14,35 @@ APPS=abstreet fifteen_min osm_viewer santa widgetry_demo
 
 TSC=npx tsc --lib dom,es2020 --module es2020 --target es6 --strict --noEmitOnError --outDir build
 
-server:
-	cd build/dist && python3 -m http.server $(SERVER_PORT)
-
 clean:
 	rm -fr build
 	rm -fr src/*/wasm_pkg
 
-# Build all of our apps
-build: $(APPS) build/dist/index.html
```

Comment:
just moved some things around, hoping to have the things you're likely to care about towards the top